src/server/socket.ts

```diff
@@ -125,6 +125,18 @@ export function bindListeners(io: socketio.Server, room: socketio.Namespace) {
         callback(question);
       });
 
+      socket.on("QuestionRemove", async ({ questionId }, callback) => {
+        // Remove question
+        const [question] = await knex("Question").where({ id: questionId });
+
+        if (question) {
+          await knex("Question").where({ id: question.id }).delete();
+          // Remove question from all viewers
+          io.of(`/rooms/${roomName}`).emit("QuestionRemoved", question);
```

Comment:
@KatieMacalintal Since the intent is to remove the question, can/should we just send the id as the message payload instead of sending the whole question again?